packages/airnode-deployer/src/handlers/aws/index.ts

```diff
@@ -61,5 +83,6 @@ export async function testApi(event: any) {
     return { statusCode: 400, body: JSON.stringify({ error: err.toString() }) };
   }
 
-  return { statusCode: 200, body: JSON.stringify(result) };
+  // NOTE: We do not want the user to see {"value": <actual_value>}, but the actual value itself
+  return { statusCode: 200, body: JSON.stringify(result!.value) };
```

Comment:
@amarthadan I've changed the output of testing gateway. I think this is not a big deal (major change) though since this output is only for testing